src/glmfit.jl

```diff
@@ -521,6 +526,10 @@ const FIT_GLM_DOC = """
     for a list of built-in links).
 
     # Keyword Arguments
+    - `dropcollinear::Bool=true`: Controls whether or not `lm` accepts a model matrix which
+      is less-than-full rank. If `true` (the default), only the first of each set of
+      linearly-dependent columns is used. The coefficient for redundant linearly dependent columns is
+      `0.0` and all associated statistics are set to `NaN`.
```

Comment:
```suggestion
      is less-than-full rank.
      If `true` (the default) the coefficient for redundant linearly dependent columns is
      `0.0` and all associated statistics are set to `NaN`.
      Typically from a set of linearly-dependent columns is the last ones are identified as redundant
      (however, the exact selection of columns identified as redundant is not guaranteed).
```
@nalimilan - per our discussion, maybe we should change the docstring as I propose here?